apps/safe-claiming-app/src/config/theme.ts

```diff
@@ -1,173 +1,182 @@
+        fontWeight: "bold",
+      },
+      subtitle1: {
+        fontStyle: "normal",
+        fontWeight: 400,
+        fontSize: "16px",
+        lineHeight: "24px",
+      },
+      subtitle2: {
+        fontStyle: "normal",
+        fontWeight: 400,
+        fontSize: "14px",
+        lineHeight: "24px",
+        color: colors.text.secondary,
+      },
+      caption: {
+        fontSize: "12px",
+        lineHeight: "16px",
+        letterSpacing: "0.4px",
+      },
     },
-    warning: {
-      main: "#FF8061",
+    shape: {
+      borderRadius: 8,
     },
-  },
-  shape: {
-    borderRadius: 8,
-  },
-  components: {
-    MuiTooltip: {
-      styleOverrides: {
-        tooltip: {
-          backgroundColor: "#121312",
-          color: "#FFF",
-          fontSize: 12,
-          paddingTop: "12px",
-          paddingBottom: "8px",
-          paddingLeft: "16px",
-          paddingRight: "16px",
-          maxWidth: "275px",
-          boxShadow: "rgba(33, 48, 77, 0.1) 0px 0px 10px 0px",
-        },
-        arrow: {
-          color: "#121312",
+    components: {
+      MuiCard: {
+        styleOverrides: {
+          root: ({ theme }) => ({
+            borderRadius: theme.shape.borderRadius,
+            background: theme.palette.background.paper,
+          }),
         },
       },
-    },
-    MuiPaper: {
-      styleOverrides: {
-        elevation: {
-          zIndex: 2,
-          color: "#121312",
+      MuiTooltip: {
+        styleOverrides: {
+          tooltip: {
+            backgroundColor: colors.background.default,
+            color: colors.text.primary,
+            fontSize: 12,
+            paddingTop: "12px",
+            paddingBottom: "8px",
+            paddingLeft: "16px",
+            paddingRight: "16px",
+            maxWidth: "275px",
+            boxShadow: "rgba(33, 48, 77, 0.1) 0px 0px 10px 0px",
+          },
+          arrow: {
+            color: colors.text.primary,
+          },
         },
       },
-    },
-    MuiDivider: {
-      styleOverrides: {
-        light: {
-          borderColor: "#EEEFF0",
+      MuiPaper: {
+        styleOverrides: {
+          elevation: {
+            zIndex: 2,
+            color: colors.text.primary,
+          },
         },
       },
-    },
-    MuiButton: {
-      styleOverrides: {
-        sizeLarge: {
-          padding: "10px 40px",
+      MuiDivider: {
+        styleOverrides: {
+          light: {
+            borderColor: "#EEEFF0",
+          },
         },
-        contained: {
-          border: "1px solid transparent",
-          "&:hover": {
-            backgroundColor: "transparent",
-            border: "1px solid #121312",
-            color: "#121312",
+      },
+      MuiButton: {
+        styleOverrides: {
+          sizeLarge: {
+            padding: "10px 40px",
+          },
+          contained: {
+            border: "1px solid transparent",
+            "&:hover": {
+              backgroundColor: "transparent",
```

Comment:
This seems different to the buttons in web-core where it keeps the background color on hover and just makes it a bit darker. I would suggest keeping it consistent.